liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -164,9 +174,19 @@ void SourceReferenceFormatter::printSourceLocation(SourceReference const& _ref)
 void SourceReferenceFormatter::printExceptionInformation(SourceReferenceExtractor::Message const& _msg)
 {
 	// exception header line
-	errorColored() << _msg.severity;
+	errorColored() << _msg.severity_error;
+    warningColored() << _msg.severity_warning;
+    infoColored() << _msg.severity_info;
+
```

Comment:
You're mixing tabs and spaces here. Please note that we indent C++ code with tabs.

Reply:
Okay will correct it 👍